examples/chart/teleport-cluster/templates/proxy/service.yaml

```diff
@@ -36,12 +36,13 @@ spec:
 {{- with $proxy.service.spec }}
   {{- toYaml . | nindent 2 }}
 {{- end }}
+{{- if and (not .Values.service.skipPorts) (empty .Values.service.spec.ports) }}
```

Comment:
`.Values` must not be used directly when `$proxy` is available as this contains the result of the generic `.Values` and the proxy-specific values `.Values.proxy`